NashTeamAlpha/CustomerIncidentPortal
# wrote tests and implemented code for saving incidents, listing incide…

## Pull Request Description

Include the applicable information from the list below here 

1. Tested whether or not incidents can be saved to the database.
1. Incidents can be created with customer first and last name.
1. Tested that an incident could be created.

## Steps to Test

Include the applicable information from the list below here

1. Git fetch
1. Switch to branch jd-incidentTest
1. Make sure all tests pass (restart VS if any fail)

## Link to Feature Ticket

Provide the issue number that this pull request fixes
#31, #32, #3
